specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/preview/2021-02-01-preview/examples/DPSGetOperationResult.json

```diff
@@ -0,0 +1,17 @@
+{
+  "parameters": {
+    "provisioningServiceName": "myFirstProvisioningService",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "91d12660-3dec-467a-be2a-213b5544ddc0",
+    "operationId": "MTY5OTNmZDctODI5Yy00N2E2LTkxNDQtMDU1NGIyYzY1ZjRl",
+    "asyncinfo": "1508265712453"
+  },
+  "responses": {
+    "200": {
+      "body": {
```

Comment:
If this is the response from the URI specified in the Location header, it needs to return the exactly same response as that would have been returned had the operation been executed synchronously.

https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#202-accepted-and-location-headers

Reply:
Temporarily resolved. See details in this [comment](https://github.com/Azure/azure-rest-api-specs/pull/12378#discussion_r560402312)